src/compliance/industry-compliance.md

```diff
@@ -0,0 +1,11 @@
+---
+group: compliance
+title: Industry Compliance
+---
+
+Merchants are obligated to meet industry guidelines for maintaining a secure environment, legal requirements in their jurisdiction, and best practices for online commerce. To learn more, see the [Industry Compliance]{https://docs.magento.com/m2/ee/user_guide/stores/compliance-industry.html) section in the Magento User Guide.
```

Comment:
```suggestion
Merchants are obligated to meet industry guidelines for maintaining a secure environment, legal requirements in their jurisdiction, and best practices for online commerce. To learn more, see the [Industry Compliance](https://docs.magento.com/m2/ee/user_guide/stores/compliance-industry.html) section in the Magento User Guide.
```

Reply:
The suggested change inserts underscores "_Magento User Guide_".  I think it's OK as is.

Reply:
I don't recall making a suggestion here. The only thing that looks weird is the two `<span>` classes. What are those? We don't use inline spans like that in devdocs.

Reply:
yeah, I saw those. I don't know where they came from, but I took them out!

Reply:
Fixed

Reply:
@ccondra, when I look at your branch locally, I still see an error, which reminds me of why I originally commented on this. the link syntax is wrong:

```
[Industry Compliance]{https://docs.magento.com/m2/ee/user_guide/stores/compliance-industry.html)
```

Should be:

```
[Industry Compliance](https://docs.magento.com/m2/ee/user_guide/stores/compliance-industry.html)
```

Reply:
I put the underscore to italicize the name of a user guide -- based on style adapted in the What's New content.  See https://developers.google.com/style/typographical-conventions.

Reply:
OK, that makes sense. I'll put underscores back to italicize the name of the user guide. It was confusing because it got tangled up with the span tag that was inserted because of the hyphen at the end of the link (which we do in merchdocs). I got it!